### DISCUSSION TITLE
createQuery does not throw any error when bind call has incorrect parameter number

### DISCUSSION THREAD

**Discussion by meta-forte** [jdbi/jdbi#2646] [Q&A]
`createQuery` is not throwing errors or emitting warning logs when I pass the wrong parameter Index in the `bind` call

````Kotlin
  it.createQuery("select a.* from Table1 a where a.name=?").bind(1,"Tokyo").firstOrNull()

````
 The bind call does not indicate that the first param was not bound

**Comment by stevenschlansker** [comment]
Thanks for reporting this. I will make this an issue and we can see if it is possible to improve error detection here.
